definitions/ChrCustomizationConversion.dbd

```diff
@@ -2,19 +2,19 @@ COLUMNS
 int ID
 int<ChrRaces::ID> ChrRacesID?
 int Sex?
-int UICustomizationType? // 1 = Skin Color, 2 = Face, 3 = Hair Style, 4 = Hair Color, 5 = Features
-int Index?
+int OptionID? // 1 = Skin Color, 2 = Face, 3 = Hair Style, 4 = Hair Color, 5 = Features, 6 = CustomDisplayOption1, 7 = CustomDisplayOption2, 8 = CustomDisplayOption3
```

Comment:
This should have been an "official name" from Lua as well?

Reply:
I haven't found references to this table in Lua code, only a single assertion message

Reply:
Hm, yeah, I can't find it either anymore and my commit doesn't mention the source. Sad. Oh well. :shrug: